src/show.jl

```diff
@@ -118,44 +118,44 @@ _tree_icon(obj::Attributes) = _tree_icon(obj.parent)
 _tree_head(io::IO, obj) = print(io, _tree_icon(obj), " ", obj)
 _tree_head(io::IO, obj::Datatype) = print(io, _tree_icon(obj), " HDF5.Datatype: ", name(obj))
 
-function _tree_children(parent::Union{File, Group}, attributes::Bool)
+function _tree_children(parent::Union{File, Group}, show_attributes::Bool)
```

Comment:
@jmert  attributes was clashing with the `attributes` method so I had to rename it.